engine/script/src/script_table.cpp

```diff
@@ -26,17 +26,6 @@ extern "C"
 #include <lua/lualib.h>
 }
 
-// Emscripten requires data access to be aligned and setting __attribute__((aligned(X)) should force emscripten to
-// make multiple access instructions and merge them.
-// However, there's a bug in emscripten which causes __attribute__((aligned(X)) to sometimes fail
-// To work around this, we can use a typedef
-// The bug is tracked: https://github.com/kripken/emscripten/issues/2378
-#ifdef __EMSCRIPTEN__
-typedef lua_Number __attribute__((aligned(4))) lua_Number_4_align;
-#else
-typedef lua_Number lua_Number_4_align;
-#endif
```

Comment:
Replacing the code with a memcpy and a union should work fine.
The https://github.com/emscripten-core/emscripten/issues/2378 was reported in 2014, and even though it was closed due to it being stale, I don't suspect this change will cause any issues.